config/main.py

```diff
@@ -506,21 +506,23 @@ def save(filename):
 @config.command()
 @click.option('-y', '--yes', is_flag=True)
 @click.argument('filename', default='/etc/sonic/config_db.json', type=click.Path(exists=True))
-def load(filename, yes):
-    """Import a previous saved config DB dump file."""
+@click.argument('init_cfg_file', default='/etc/sonic/init_cfg.json', type=click.Path(exists=True))
+def load(filename, init_cfg_file, yes):
+    """Import previous saved config DB dump file and init_cfg.json."""
     if not yes:
-        click.confirm('Load config from the file %s?' % filename, abort=True)
-    command = "{} -j {} --write-to-db".format(SONIC_CFGGEN_PATH, filename)
+        click.confirm('Load config from the files %s and %s?' % (filename, init_cfg_file), abort=True)
+    command = "{} -j {} -j {} --write-to-db".format(SONIC_CFGGEN_PATH, filename, init_cfg_file)
     run_command(command, display_cmd=True)
```

Comment:
On second thought, I think we should not load init_cfg.json during `config load`, as this command is meant to load the contents of config_db.json over the existing running config. Since the init_cfg.json should not change until the device is upgraded, there should be no need to load init_cfg.json here. Please undo these changes.

Reply:
Undo the changes.